carapace-server/src/main/java/org/carapaceproxy/api/CertificatesResource.java

```diff
@@ -62,6 +74,18 @@ public String getHostname() {
             return hostname;
         }
 
+        public boolean isDynamic() {
+            return dynamic;
+        }
+
+        public DynamicCertificateState getStatus() {
```

Comment:
Do not expose business logic beans on the API.
Please create a bean (inner static public class here:

Reply:
Maybe for the frontend it could be simply a string

Reply:
Sure

Reply:
I meant something different.
If you call toString() the API would break if someone adds/remove values and/or changes the toString() method.
You have to explicitly transcode the values. a String is fine but not this way